tests/functional_tests/ffront_tests/test_unrollPowOp.py

```diff
@@ -0,0 +1,139 @@
+import pathlib
+
+from eve.pattern_matching import ObjectPattern
+from functional.ffront import common_types as ct, field_operator_ast as foast
+from functional.ffront.func_to_foast import UnrollPowerOp
+
+
+def _make_power_testee(pow_n: str):
+    _set_source_location = foast.SourceLocation(
+        line=106, column=12, source=str(pathlib.Path(__file__).resolve())
+    )
```

Comment:
```suggestion
    loc = foast.SourceLocation(
        line=0, column=0, source="none")
    )
```
Don't use verbs like `set` in variable names. Prepending them by an underscore is additionally reserved for variables and functions that are private (in Python in the sense of a convention to not use it outside).